site/feed.njk

```diff
@@ -9,23 +9,21 @@ pagination:
 ---
 <?xml version="1.0" encoding="utf-8"?>
 <feed xmlns="http://www.w3.org/2005/Atom">
+  {% set absoluteFeedUrl %}{{ site.url }}{% if feed.url %}{{ feed.url | url }}{% endif  %}{% endset %}
   <title>{{ feed.title or site.title }}</title>
-  <subtitle>{{ feed.description or site.summary }}</subtitle>
-  <link href="{{ site.url }}{{ feed.permalink }}" rel="self" />
-  <link href="{{ site.url }}/" />
+  <subtitle>{{ feed.description or site.description }}</subtitle>
+  <link href="{{ absoluteFeedUrl }}" rel="self"/>
+  <link>{{ site.url }}{{ feed.permalink | url }}</link>
```

Comment:
taking a look at an rss validator, it seemed to indicate that link was supposed to have an href

Reply:
this is the one i was using https://validator.w3.org/feed/#validate_by_input